service/devdoc/registry_requirements.md

```diff
@@ -370,17 +370,234 @@ Request-Id: <guid>
 ```
 **]**
 
+## Module Twins
 
-### All HTTP requests
+### getModuleTwin(deviceId: string, moduleId: string, done: Registry.ResponseCallback): void;
```

Comment:
>Registry.ResponseCallback [](start = 60, length = 25)

I know it wasn't the case with getTwin but it'd be nice to have more strongly typed callback parameters here.